app/views/handlers/feedback.py

```diff
@@ -123,7 +137,7 @@ def question_schema(self):
 
         options = (
             {"label": value, "value": value}
-            for value in detail_answers_option_map.get(self._schema.form_type or "H")
+            for value in detail_answers_option_map.get(self._schema.form_type or "H")  # type: ignore
```

Comment:
This ignore can be removed, it is complaining cause it is a `.get()`. It should be key access. `self._schema.form_type or "H"` will always resolve to an existing key.
```suggestion
            for value in detail_answers_option_map[self._schema.form_type or "H"]
```